### PR TITLE
[FIX] event: Apply Bootstrap 4 migration to description

### PR DESCRIPTION
**Pull request by yajo**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr
@Tecnativa

### CODE REVIEW

File: addons/event/migrations/12.0.1.0/post-migration.py [hunk 1]
```diff
@@ -1,18 +1,21 @@
 # Copyright 2018 Eficent <http://www.eficent.com>
 # Copyright 2019 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Tecnativa - Jairo Llopis
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openupgradelib import openupgrade
+from openupgradelib import openupgrade, openupgrade_120
 
 
-@openupgrade.migrate(use_env=False)
```
Comment (pedrobaeza) [inline addons/event/migrations/12.0.1.0/post-migration.py:8]: @mreficent look why I always want to have `env` by default...

File: addons/event/migrations/12.0.1.0/post-migration.py [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
-def migrate(cr, version):
+@openupgrade.migrate()
+def migrate(env, version):
     openupgrade.load_data(
-        cr, 'event', 'migrations/12.0.1.0/noupdate_changes.xml')
+        env.cr, 'event', 'migrations/12.0.1.0/noupdate_changes.xml')
```
Comment (pedrobaeza) [inline addons/event/migrations/12.0.1.0/post-migration.py]: @Yajo for reducing diff, put a first line with `cr = env.cr` and don't touch the rest

File: addons/event/migrations/12.0.1.0/post-migration.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply addons/event/migrations/12.0.1.0/post-migration.py:8]: yeah, from some time ago, I do always do `env` by default...

File: addons/event/migrations/12.0.1.0/post-migration.py (hunk 2 above)
Comment (yajo) [reply addons/event/migrations/12.0.1.0/post-migration.py]: Is it worth it to add a variable just for the sake of reducing diff? :roll_eyes:

Comment (pedrobaeza) [reply addons/event/migrations/12.0.1.0/post-migration.py]: Yes, and check Travis that is red

**Comment by MiquelRForgeFlow** [comment]
This PR makes the branch 12.0 **red** due to https://github.com/OCA/openupgradelib/pull/160. It seems the `convert_field_bootstrap_3to4` needs a fine tuning.

```xml
2019-07-24 12:19:54,072 6677 ERROR openupgrade OpenUpgrade: Error converting string BS3 to BS4:
<section class="s_text_block_image_fw oe_img_bg" style="background-position: 50% 100%; background-image: url(/web/image/website.s_background_image_02);">
   <div class="container">
      <div class="row">
         <div class="bg-black-25 col-md-9 mb64 mt64">
            <h2 style="font-size:62px">5 Intensive Days</h2>
            <h4 class="text-muted"><font style="color: rgb(255, 255, 255); font-size: 24px;">to learn <i>Odoo Programming</i></font></h4>
         </div>
      </div>
   </div>
</section>
<div class="container">
<div class="row">
<div class="col-md-12 text-center mb8 mt0">
<h2>Course Summary</h2>
</div>
<div class="col-md-12 mt16 mb0">
<p><span style="text-align: -webkit-center; ">This course is dedicated to developers who need to grasp knowledge of the <strong>business applications development </strong>process. This course is for new developers or for IT professionals eager to learn more about technical aspects.</span></p>
</div>
</div>
</div>
<div class="container">
<div class="row">
<div class="col-md-12 text-center mt0 mb16">
<h2>What you will learn?</h2>
</div>
<div class="col-md-4">
<div class="panel panel-info">
<div class="panel-heading text-center">
<h2 style="margin: 0">Day 1</h2>
<p class="text-muted" style="margin: 0">Introduction to Javascript</p>
</div>
<ul class="list-group"><li class="list-group-item">Hello World</li>
	<li class="list-group-item">Variables &amp; Operators</li>
	<li class="list-group-item">Dive into Strings</li>
	<li class="list-group-item">Functions</li>
	<li class="list-group-item">Loops</li>
	<li class="list-group-item">Arrays</li>
</ul><div class="panel-footer text-center">
<p class="text-muted"><i>You will be able to develop simple dynamic components in HTML pages.</i></p>
</div>
</div>
</div>
<div class="col-md-4">
<div class="panel panel-info">
<div class="panel-heading text-center">
<h2 style="margin: 0">Day 2</h2>
<p class="text-muted" style="margin: 0">Odoo Web Client</p>
</div>
<ul class="list-group"><li class="list-group-item">Introduction to JQuery</li>
	<li class="list-group-item">Advanced JQuery</li>
	<li class="list-group-item">Underscore</li>
	<li class="list-group-item">Introduction to QWeb</li>
	<li class="list-group-item">Controllers and Views</li>
	<li class="list-group-item">Bootstrap CSS</li>
	<li class="list-group-item">Calling the ORM</li>
</ul><div class="panel-footer text-center">
<p class="text-muted"><i>You will be able to create dynamic page interacting with the ORM.</i></p>
</div>
</div>
</div>
<div class="col-md-4">
<div class="panel panel-info">
<div class="panel-heading text-center">
<h2 style="margin: 0">Day 3</h2>
<p class="text-muted" style="margin: 0">Building a Full Application</p>
</div>
<ul class="list-group"><li class="list-group-item">Modules</li>
	<li class="list-group-item">Python Objects</li>
	<li class="list-group-item">Report Engine</li>
	<li class="list-group-item">Workflows</li>
	<li class="list-group-item">Training Center Module</li>
	<li class="list-group-item">Integrated Help</li>
	<li class="list-group-item">How to Debug</li>
</ul><div class="panel-footer text-center">
<p class="text-muted"><i>You will be able to develop a full application with backend and user interface.</i></p>
</div>
</div>
</div>
</div>
</div>
<div class="container">
<div class="row">
<div class="col-md-12 text-center mb16 mt0">
<h2>Requirements</h2>
</div>
<div class="col-md-12 mb16 mt16">
<p><strong>Objectives:</strong></p>
<p><strong>Having attended this course, participants should be able to:</strong></p>
<ul><li>Understand the development concepts and architecture;</li>
	<li>Install and administer your own server;</li>
	<li>Develop a new module for a particular application.</li>
</ul><p></p>
<p><strong>Our prices include:</strong></p>
<ul><li>drinks and lunch;</li>
	<li>training material.</li>
</ul><p></p>
<p><strong>Requirements:</strong></p>
<ul><li>Bring your own laptop.</li>
	<li>Participants are expected to have some knowledge in programming. A basic knowledge of the Python programming is recommended.</li>
	<li>Participants preferably have a functional knowledge of our software (see Functional Training).</li>
</ul><p></p>
<p>To get more information, visit the <a target="_blank" href="http://odoo.com/">Odoo Official Website</a>.</p>
</div>
</div>
</div>
<div class="container oe_dark">
<div class="row">
<div class="col-md-12">
<h1 class="text-center">Read Great Contents</h1>
<h3 class="text-muted text-center">What do people say about this course?</h3>
</div>
<div class="mt16 mb32 col-md-6">
<blockquote data-o-mail-quote-node="1" data-o-mail-quote="1">
<p>I did not expect such a great learning experience. I feel like I can develop anything now.</p>
<small>John Doe, CEO</small></blockquote>
</div>
<div class="mt16 mb32 col-md-6">
<blockquote data-o-mail-quote-node="1" data-o-mail-quote="1">
<p>This course help me build my first application within a month. Definetly worth its price.</p>
<small>John Doe, CEO</small></blockquote>
</div>
</div>
</div>
 
2019-07-24 12:19:54,072 6677 ERROR openupgrade OpenUpgrade: event: error in migration script event/migrations/12.0.1.0/post-migration.py: 'lxml.etree._Attrib' object has no attribute 'copy' 
2019-07-24 12:19:54,072 6677 ERROR openupgrade OpenUpgrade: 'lxml.etree._Attrib' object has no attribute 'copy' 
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/openupgradelib/openupgrade.py", line 1502, in wrapped_function
    if use_env2 else cr, version)
  File "/home/travis/build/OCA/OpenUpgrade/addons/event/migrations/12.0.1.0/post-migration.py", line 22, in migrate
    env, "event.event", "description")
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/openupgradelib/openupgrade_120.py", line 366, in convert_field_bootstrap_3to4
    env, model_name, field_name, domain,
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/openupgradelib/openupgrade_120.py", line 390, in _convert_field_bootstrap_3to4_orm
    lambda old, *a, **k: convert_string_bootstrap_3to4(old),
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/openupgradelib/openupgrade.py", line 2004, in update_field_multilang
    new_value = method(record[field], lang_code, record)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/openupgradelib/openupgrade_120.py", line 390, in <lambda>
    lambda old, *a, **k: convert_string_bootstrap_3to4(old),
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/openupgradelib/openupgrade_120.py", line 343, in convert_string_bootstrap_3to4
    html_string, ALL_REPLACEMENTS, pretty_print,
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/openupgradelib/openupgrade_tools.py", line 91, in convert_html_fragment
    convert_xml_node(node, **instructions)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/openupgradelib/openupgrade_tools.py", line 170, in convert_xml_node
    "attrs": node.attrib.copy(),
AttributeError: 'lxml.etree._Attrib' object has no attribute 'copy'
2019-07-24 12:19:54,074 6677 ERROR openupgrade odoo.modules.loading: Error executing post migration script for module event
: 'lxml.etree._Attrib' object has no attribute 'copy' 
```

@Yajo can you provide the solution, ASAP?

**Pull request closed without merging**